### ISSUE TITLE
run_ensemble doesn't work without temperature observations

### ISSUE THREAD

**Issue by jorritmesman** [aemon-j/LakeEnsemblR#139]
If you try to run run_ensemble without specifying a temperature observation file in the master config file, the following error is created. 

```
Loading temperature observations...
Error in read.table(file = file, header = header, sep = sep, quote = quote,  : 
  no lines available in input
In addition: Warning message:
In file(file, "rt") :
  file("") only supports open = "w+" and open = "w+b": using the former
```

It should be possible to run without observations (if you specified an initial profile)

**Comment by jorritmesman** [comment]
You can avoid the above error by typing `NULL` in the master config file where the file should be specified. However, then obs_deps is set to NULL internally, which generates an error when get_output is used to extract output from the model run. 

If no observations are provided, obs_deps should be set to the output depths requested by the user.

Typing `NULL` is not the ideal solution, and a different solution (i.e. leaving it empty) is used elsewhere in the master config file.

**Comment by jorritmesman** [comment]
Fixed in PR #141. Now you can both type NULL or leave it open.
